kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/server/net2/configuration/NetworkConfigurationServicePropertiesBuilder.java

```diff
@@ -205,8 +208,12 @@ private void setWifiInfraProperties() {
             if (GwtServerUtil.PASSWORD_PLACEHOLDER.equals(gwtWifiConfig.getPassword())
                     && oldGwtNetInterfaceConfig instanceof GwtWifiNetInterfaceConfig) {
                 GwtWifiNetInterfaceConfig gwtWifiNetInterfaceConfig = (GwtWifiNetInterfaceConfig) oldGwtNetInterfaceConfig;
-                this.properties.setWifiInfraPassphrase(this.ifname,
-                        gwtWifiNetInterfaceConfig.getStationWifiConfig().getPassword());
+                gwtWifiNetInterfaceConfig.setUnescaped(true);
+                
+                GwtWifiConfig gwtStationConfig =  gwtWifiNetInterfaceConfig.getStationWifiConfig();
+                gwtStationConfig.setUnescaped(true);
```

Comment:
Just to keep things consistent I used the same setUnescaped method used before.  Would be less code to directly use the unescape function but feel like this is better for future maintainability.